dracut-init.sh

```diff
@@ -255,6 +255,21 @@ inst() {
     fi
 }
 
+inst_recur() {
+    local x
+    for x in "$dracutsysrootdir${1%/}"/*; do
+        x=${x#"$dracutsysrootdir"}
+        if [[ -d "$dracutsysrootdir$x" ]]; then
+            inst_dir "$x"
+            inst_recur "$x"
+        elif [[ -f "$dracutsysrootdir$x" ]]; then
+            inst "$x"
+        else
+            break
```

Comment:
This should never happen, right? If it ever happens, you probably want to issue a warning instead...

Reply:
Btw. in general, isn't `inst_dir` implicitly recursive? What is `inst_recur` exactly solving?